packages/components/fields/date-field/README.md

```diff
@@ -102,7 +102,9 @@ export default Example;
 ### Signature `onInfoButtonClick`
 
 ```ts
-() => void
+(
+  event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
+) => void
```

Comment:
should this be updated for the other fields as well?

Reply:
If they're off, they should. But this README files are generated based on the component types, and I'd like to fix the types in an upcoming round after we achieved feature parity. 

I only fixed this one cause @ddouglasz  mentioned it was different in the v5 version, but now after checking, it seems he must have slipped to the DateTimeField Component when comparing as the v5 version already had the same issue.

I know it's 💩 that the components are not 100% correctly typed, but when doing the conversion of the stories I simply can't question every type that I encounter, I have to assume they are correct. Only If storybook had issues parsing them I took a closer look and fixed things when it felt necessary.

Reply:
Then it's definitely out of scope for this PR, thanks!